pkg/redshift

```diff
@@ -0,0 +1,23 @@
+[mirrors]
+https://github.com/jonls/redshift/archive/b210813c7ffdd0eae75eb60c7bbbb62b6b109ebe.tar.gz
+
+[vars]
+tarball=b210813c7ffdd0eae75eb60c7bbbb62b6b109ebe.tar.gz
```

Comment:
:D the tarball directive allows you to save the tarball under a proper name, e.g. redshift-git-b210813.tar.gz or so (that helps identifying the tarball)